### DISCUSSION TITLE
Add an option to disable the intro sequence

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20748] [Ideas]
All of the intro sequences are loud, slow, and annoying (in my personal opinion), there should be an option in the intro sequence dropdown to just choose "none"
I don't know how long it actually takes for the game assets to load, but considering they are loaded by the time the intro is done, I can assume the intro sequence is all that is preventing the game from taking less time to start (and be interactable), so disabling the intro sequence could help with that.

**Comment by nagi-desuuu** [comment]
The intro sequence is what makes osu, well, osu!
Without the iconic intros that everyone has come to love, would it really be the same old osu that you know and love?

**Reply by Doksha** [reply]
the osu stable startup time (for me, from clicking the app to being able to interact) is about 5.5 seconds
The osu lazer startup time (for me, from clicking the app to being able to interact) is about 17 seconds, more than double stable
I'd guess the majority of that extra time is due to the intro.
especially when trying to do something quickly, waiting that long for the client to start is a bit stupid

**Reply by peppy** [reply]
Please get out a stopwatch. The majority of the time is not surprisingly the "disclaimer" screen, which will eventually be removed.

The intro sequences are times to be the same as stable's. To the millisecond if you use one of the old ones.

**Comment by frenzibyte** [comment]
If you don't like the new intro, you can choose back the classic "Welcome" intro sequence:

<img width="538" alt="CleanShot 2022-10-13 at 22 30 22@2x" src="https://user-images.githubusercontent.com/22781491/195692174-5417a9e2-ff44-46e6-807b-3c58776a0975.png">

If you find them to be loud, you can disable both the intro theme and the "welcome" sound effect in settings.

![CleanShot 2022-10-13 at 22 30 45@2x](https://user-images.githubusercontent.com/22781491/195692224-cd2d6305-a5b4-4f30-9232-846caa51713b.png)

Other than that, I'm not sure how you find the intro sequence annoying unless you're starting and closing the game every 5 seconds.

**Reply by Doksha** [reply]
when reporting bugs I need to get the version number, (which I easily forget) also grabbing skins since they arent stored in a folder, also grabbing maps since they arent stored in a folder. There are many reasons to start the client to do just one thing, and waiting for the whole intro to play is a little annoying

**Reply by agatemosu** [reply]
A "Watch intro sequence once a day" option would be nice
Or a dropdown with "Always/Once a day/Never"

**Reply by frenzibyte** [reply]
I may agree with a separate shortened "Welcome back" intro sequence when starting the game after a short period of closing it, but all mentioned reasons sound like edge cases to me, eventually you won't need to do any of that. And all that aside, why not just use "Welcome" then and call it a day.

**Reply by Doksha** [reply]
putting all that aside, why does it need to take over 16 seconds to start? Does lazer have so much more stuff that it just takes longer to load everything, or is that all part of the intro (ie the "this is lazer" screen)
Does it _need_ to take so long, or is it just intentionally slow?
Why does it have to be forced on the user to have the game take so long to load?

Did peppy spend a ton of money on the intros or something? because it doesnt really make sense.

**Reply by frenzibyte** [reply]
Please take that attitude of the last sentence elsewhere. For me, the game starts at about 13 seconds, 4 seconds for the window to pop up, 6 seconds is spent on the disclaimer screen, while only 3 seconds are spent on the intro itself. 

Now, I don't follow what you're requesting here, the "this is lazer" screen is not part of the intro sequence that you're requesting to be disabled. If that's what you've been annoyed by all along, you *could* have just mentioned that instead..?

**Reply by Doksha** [reply]
Im just confused as to why they need to be forced on the user. At least have something like trackmania where you can click to skip the disclaimer/intro. The question I asked was a legitimate question, I did not intend to insult anyone.
